### ISSUE TITLE
need a better way to import sass files

### ISSUE THREAD

**Issue by mgiraldo** [NYPL/design-toolkit#13]
a bunch of files are importing the same things and i'm not sure if that is proper. ideas?

**Comment by EdwinGuzman** [comment]
Two possibilities I can think of right now.
1. CDN could work but we'd have to work with IT to get a system going to deploy files so that they can live in the nypl.org domain.
2. npm module. This requires webpack and other small configuration, but, with the existing code structure in this repo and assuming it's called `@nypl/design-toolkit`, importing would be:
    ```
    // main.scss
    @import '~@nypl/design-toolkit/sass/_forms.scss';
    @import '~@nypl/design-toolkit/sass/_grid.scss';
    ```

**Comment by mgiraldo** [comment]
i don't think we need to go that far. more like a SASS-friendly including pattern. GOV.uk [doesn't](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_grid_layout.scss) [seem](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_typography.scss) [to worry](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_colours.scss) about multiple inclusion, though.

**Comment by EdwinGuzman** [comment]
Sounds good. If those are too many includes they can be grouped together.
```
    @import '~@nypl/design-toolkit/sass/_base.scss';
```
and `_base.scss` would include `_forms.scss`, `_grid.scss`, etc. One include for multiple files.

**Comment by rhernand3z** [comment]
I am playing around with creating an EyeGlass module to import sass files similar to the Neat SASS framework.

References:

- http://eyeglass.rocks/

**Comment by mgiraldo** [comment]
fixed in bcb283ccc9ca50bdbe52d8857f84276f64e53f49
